common/patches/dont-crash-if-no-test-charm-repo-is-there.diff

```diff
@@ -0,0 +1,18 @@
+The regular juju test code sets up a test charm repository that
+fake-jujud doesn't have access to at run time. So we set the
+test charm repository to be empty.
+
+Index: 2.0.2/src/github.com/juju/juju/testcharms/charm.go
+===================================================================
+--- 2.0.2.orig/src/github.com/juju/juju/testcharms/charm.go
++++ 2.0.2/src/github.com/juju/juju/testcharms/charm.go
+@@ -17,7 +17,9 @@ import (
+ )
+ 
+ // Repo provides access to the test charm repository.
++// XXX fake-juju: start patch
+ var Repo = testing.NewRepo("charm-repo", "quantal")
++///XXX fake-juju: end patch
```

Comment:
Ugh! Good catch, fixed it to be:

```
-var Repo = testing.NewRepo("charm-repo", "quantal")
+// XXX fake-juju: start patch
+var Repo = &testing.Repo{}
+// XXX fake-juju: end patch
```

like it is currently in juju-core.patch.